crates/resolver/src/config.rs

```diff
@@ -293,29 +293,6 @@ impl Default for ResolverConfig {
     }
 }
 
-/// a compatibility wrapper around rustls
-/// ClientConfig
-#[cfg(feature = "dns-over-rustls")]
-#[derive(Clone)]
-pub struct TlsClientConfig(pub Arc<ClientConfig>);
-
-#[cfg(feature = "dns-over-rustls")]
-impl std::cmp::PartialEq for TlsClientConfig {
```

Comment:
I can't remember why we needed this exactly, I'm guessing as an internal part of the NameserverConfig?

Reply:
Yes, I guess so too.